### ISSUE TITLE
Is there a way to turn off warnings for non-defined keys?

### ISSUE THREAD

**Issue by spherop** [alibaba/react-intl-universal#45]
I am wondering if it's possible to turn off the browser console warnings for missing keys in a locale file?

eg in chrome console:

react-intl-universal key "ZIP_CODE" not defined in en-US
tracker.js:13 react-intl-universal key "REDEEM_PROMO_CODE" not defined in en-US
etc.

This is causing a lot of log pollution.

Thanks.

**Comment by cwtuan** [comment]
why not just define a key to fix it?

**Comment by spherop** [comment]
@cwtuan - good question - here's why:

Since the site is in English, and the developers speak English natively, made sense to write a little helper that creates the translations from the English text - and use the defaultMessage mechanism to fallback.

```
export const t = (stringToTranslate, defaultString = false, options = null) => {
  if (!stringToTranslate) { return null; }
  defaultString = defaultString || stringToTranslate;
  return intl.get(stringToTranslate.split(' ').join('_').toUpperCase(), options).defaultMessage(defaultString);
};
```

In render we have:

```
<span className="titleRow">{t('Thank you')}, {this.props.first_name}!</span>
<span className="titleRow">{t('Your booking is confirmed')}!</span>
```

Thus `t('Thank you')` becomes `intl.get('THANK_YOU').defaultMessage('Thank you')`

Therefore, we don't need English versions of these because we fallback on defaultMessage. Since we have a huge amount to translate, saves us from having English versions of most translations. Also, keeps our views readable with the real text vs harder to grok keys.

Hope this all makes some sense... or maybe I am making an odd move here?

**Comment by spherop** [comment]
This pr would resolve: #44

**Comment by Panoplos** [comment]
I second this. During development, it is annoying to have so many warnings.

**Comment by cwtuan** [comment]
Fixed v1.11.1.
https://github.com/alibaba/react-intl-universal/releases/tag/1.11.1
